ci-operator/config/stolostron/policy-collection/stolostron-policy-collection-main__opp-ocp4.12-interop.yaml

```diff
@@ -55,6 +64,8 @@ tests:
     - ref: acm-policies-openshift-plus-setup
     - ref: acm-policies-openshift-plus
     - chain: cucushift-installer-check-cluster-health
+    - ref: acm-tests-clc-create
```

Comment:
Could we move the CLC  create after the `acm-policies-openshift-plus`?  I think all of the other "product" tests should run with the managed clusters at least defined.

Reply:
Yes, that's totally fine